docs/reattacher.md

```diff
@@ -0,0 +1,22 @@
+# Transaction reattacher
+
+`Transaction reattacher` is a service that helps persistent pending transactions to reattach to the Tangle. A persistent transaction is a transaction that does not be confirmed more than 30 minutes.
+
+When enabling the external database for transaction reattachment, `Tangle-Accelerator`  will store transactions issued by API `Send Transfer Message`.
+
+`Transaction reattacher` will periodically read pending transactions from a specific ScyllaDB cluster. Send queries to an IOTA full node to get the inclusion status of transactions. If the inclusion status is confirmed, update the status of the transaction. If finding persistent transactions, perform reattachment, which will do tips selection and POW for the original bundle, and reattach it to the Tangle.  After reattachment, `reattacher` will update the new transaction hash to the ScyllaDB cluster.
+
+
+See [docs/build.md](https://github.com/DLTcollab/tangle-accelerator/docs/build.md) for more information about enabling transaction reattachment.
```

Comment:
Shorten the hyperlink to `docs/build.md`.

Reply:
done.